crypto/rand/rand_lcl.h

```diff
@@ -17,21 +17,24 @@
 # include <openssl/ec.h>
 # include "internal/rand.h"
 
-/* we require 256 bits of randomness */
-# define RANDOMNESS_NEEDED (256 / 8)
+/* Amount of randomness (in bytes) we want for initial seeding. */
+# define RANDOMNESS_NEEDED              (128 / 8)
```

Comment:
just to clarify, you don't need the same number of bits of randomness as the keys your generating.  (You probably know that, but others reading this might not.)  You're not convinced, can you explain why?  Pauli is convinced.  I point to https://www.ietf.org/rfc/rfc4086.txt sec 3.1